source/iterators.tex

```diff
@@ -46,14 +46,10 @@
 called the
 \term{value type}
 of the iterator.
-All output iterators support the expression
-\tcode{*i = o}
-where
-\tcode{o}
-is a value of some type that is in the set of types that are
-\term{writable}
-to the particular iterator type of
-\tcode{i}.
+All output iterators \tcode{i} have a non-empty set of types that are
```

Comment:
I don't think the plural works here. How about "An output iterator `i` has a non-empty set..."?